### PR TITLE
fix: correct http timeout configs‘ default values and ignorance by HttpRequestNode

### PR DESCRIPTION
**Pull request by bowenliang123**
# Checklist:

> [!IMPORTANT]  
> Please review the checklist below before submitting your pull request.

- [x] Please open an issue before creating a PR or link to an existing issue
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I ran `dev/reformat`(backend) and `cd web && npx lint-staged`(frontend) to appease the lint gods

# Description

- the `min` operator causes the explicitly set http timeout configs are ignored
- replace `min` by using `Annotated[int, Field(ge=10,)]` to check http timeouts minimum
- correct default timeouts

## Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update, included: [Dify Document](https://github.com/langgenius/dify-docs)
- [x] Improvement, including but not limited to code refactoring, performance optimization, and UI/UX improvement
- [ ] Dependency upgrade

# Testing Instructions

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] annotated field with default value
- [x] annotated field with configured value

### CODE REVIEW

File: api/configs/feature/__init__.py [hunk 1]
```diff
@@ -217,18 +217,18 @@ def CONSOLE_CORS_ALLOW_ORIGINS(self) -> list[str]:
...
+    HTTP_REQUEST_MAX_CONNECT_TIMEOUT: conint(ge=10) = Field(
+        description="connect timeout in seconds for HTTP request, which should be no less than 60",
         default=300,
     )
 
-    HTTP_REQUEST_MAX_READ_TIMEOUT: NonNegativeInt = Field(
-        description="",
+    HTTP_REQUEST_MAX_READ_TIMEOUT: conint(ge=60) = Field(
+        description="read timeout in seconds for HTTP request, which should be no less than 60",
         default=600,
```
Comment (bowenliang123) [inline api/configs/feature/__init__.py]: Considering the http timeouts mainly used for code exectuion via `ssrf`, the current default value `600` is way to high for practice. 

@Yeuoly Could you confirm that `ssrf` service and caller uses this config via `httpx` in the unit of seconds?
@laipz8200 How about lower the default value to 1/10 to 60 seconds?

Comment (laipz8200) [reply api/configs/feature/__init__.py]: The default values for the timeout arguments could be:

- Connect: 10 seconds
- Read: 60 seconds
- Write: 20 seconds

File: api/configs/feature/__init__.py [hunk 2]
```diff
@@ -217,18 +217,18 @@ def CONSOLE_CORS_ALLOW_ORIGINS(self) -> list[str]:
...
     )
 
-    HTTP_REQUEST_MAX_WRITE_TIMEOUT: NonNegativeInt = Field(
-        description="",
+    HTTP_REQUEST_MAX_WRITE_TIMEOUT: conint(ge=10) = Field(
```
Comment (laipz8200) [inline api/configs/feature/__init__.py]: Please avoid using `conint` and instead set `ge` in `Field`, `conint` will be deprecated in Pydantic 3.0.

**Review by laipz8200 — CHANGES_REQUESTED** [review]

File: api/configs/feature/__init__.py (hunk 2 above)
Comment (bowenliang123) [reply api/configs/feature/__init__.py]: Resolved. Change to used `Annotated[int, Field(ge=10)]` , as suggested in official docs (https://docs.pydantic.dev/latest/api/types/#pydantic.types.conint). Thanks for the reminder.

**Review by laipz8200 — APPROVED** [review]
LGTM

**Pull request merged**